doc/quick-start/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM rust:1.67.1-slim-bullseye as builder
+
+WORKDIR /build
+
+COPY . .
+
+RUN set -eux && \
+  apt-get update && apt-get install -y git build-essential autoconf autogen libtool clang && \
+  git clone --branch v3.21.12 --recurse-submodules https://github.com/protocolbuffers/protobuf && \
+  cd protobuf && ./autogen.sh && ./configure && make -j4 && make install && cd .. && ldconfig && \
```

Comment:
Can we directly download the release of protobuf to reduce the docker build time?

Reply:
Probability not, we need to determine the CPU architecture of the host machine to download the appropriate precompiled version. Using a script may not cover all cases, so just compile from source here to make it easy.